### PR TITLE
Updated Terra node guide

### PR DESCRIPTION
**Pull request by hydrogenbond007**

### CODE REVIEW

File: docs/operations.md [hunk 1]
```diff
@@ -186,12 +183,12 @@ alerting will be documented here.
 See [Wormhole.json](../dashboards/Wormhole.json) for an example Grafana dashboard.
 
 **NOTE:** Parsing the log output for monitoring is NOT recommended. Log output is meant for human consumption and is
-not considered a stable API. Log messages may be added, modified or removed without notice. Use the metrics :-)
+not considered a stable API. Log messages may be added, modified, or removed without notice. Use the metrics :-)
```
Comment (SEJeff) [inline docs/operations.md:186]: Please don’t switch everything to an Oxford comma. This is needless churn in the docs with zero actual improvements. Fixing actual typos or grammatical errors is fine.

File: docs/operations.md [hunk 2]
```diff
@@ -186,12 +183,12 @@ alerting will be documented here.
...
 
 ## Running a public API endpoint
 
 Wormhole v2 no longer uses Solana as a data availability layer (see [design document](../whitepapers/0005_data_availability.md)).
-Instead, it relies on Guardian nodes exposing an API which web wallets and other clients can use to retrieve the signed VAA
+Instead, it relies on Guardian nodes exposing an API that web wallets and other clients can use to retrieve the signed VAA
```
Comment (SEJeff) [inline docs/operations.md:191]: Unnecessary. Please remove.

File: docs/operations.md [hunk 3]
```diff
@@ -233,18 +230,18 @@ You'll have to manage the following keys:
...
+  It's your share of the multisig mechanism that protects the Wormhole network. The guardian set can be replaced
   if a majority of the guardians agree to sign and publish a new guardian set.
 
 - A **node key**, which identifies it on the gossip network, similar to Solana's node identity or a Tendermint
   node key. It is used by the peer-to-peer network for routing and transport layer encryption.
   An attacker could potentially use it to censor your messages on the network. Other than that, it's not very
   critical and can be rotated. The node will automatically create a node key at the path you specify if it doesn't exist.
   While the node key can be replaced, we recommend using a persistent node key. This will make it easier to identify your
-  node in monitoring data and improves p2p connectivity.
+  node in monitoring data and improve p2p connectivity.
```
Comment (SEJeff) [inline docs/operations.md:241]: The original flows better. Please revert.

**Review by SEJeff — COMMENTED** [review]
Improving grammar and fixing typos is much appreciated. But please don’t make pointless changes. Non Oxford commas to Oxford commas or perfectly fine grammar to grammar that you prefer just makes for needless churn. Please don’t do this.

consider reverting those changes I suggested and you’ll have my ack. Thanks for improving wormhole.

File: docs/operations.md (hunk 1 above)
Comment (hydrogenbond007) [reply docs/operations.md:186]: Hey the only goal was to change the Terra node links but Grammarly suggested some edits thats why did it thought might be useful, sorry 😅

**Comment by hydrogenbond007** [comment]
Hey the only goal was to change the Terra node links but Grammarly suggested some edits that's why did it thought might be useful, sorry 😅

**Review by SEJeff — APPROVED** [review]
Thanks!

**Comment by hydrogenbond007** [comment]
@a5-pickle bumping up for review

**Comment by hydrogenbond007** [comment]
@evan-gray bumping for review and merge

**Comment by SEJeff** [comment]
Thinking more on this one, the Terra section is actually about Terra Classic, not Terra 2, and the links are for Terra 2. Also, this doc only mentions four chains are so when there are over 30 in wormhole. I'm thinking of closing this due to those being wrong.

What do you think @hydrogenbond007?

**Comment by hydrogenbond007** [comment]
heyo @SEJeff 
> Thinking more on this one, the Terra section is actually about Terra Classic, not Terra 2, and the links are for Terra 2

Hey the idea was intentional since, but now I could make another commit and add the distinction between terra classic and terra 2.0

> Also, this doc only mentions four chains are so when there are over 30 in wormhole.

Sorry didn't get it, could you explain it?

> I'm thinking of closing this due to those being wrong.

I think maybe I could make some more edits rather than closing since still the old links don't work and fix some other changes I realised recently.

**Pull request closed without merging**